source/ed/ed_reader.cpp

```diff
@@ -1078,8 +1079,9 @@ void EdReader::fill_vector_to_ignore(navitia::type::Data& , pqxx::work& work,
         BOOST_FOREACH(navitia::georef::edge_t e, boost::out_edges(vertex_idx, geo_ref_temp.graph)) {
             uint64_t target = boost::target(e, geo_ref_temp.graph);
             edge_to_ignore.insert({source, target});
```

Comment:
source is a "osm_idex" and target is a "node_descriptor", I can't understand how it can work. And osm_idex is a very strange name.